docs/src/pages/components/autocomplete/IntegrationDownshift.js

```diff
@@ -172,8 +175,8 @@ function DownshiftMultiple(props) {
               onChange: handleInputChange,
               onKeyDown: handleKeyDown,
               placeholder: 'Select multiple countries',
-            }),
-            label: 'Label',
+            },
+            inputProps: getInputProps(),
```

Comment:
This change seems to break the focused state. Focus the input, blur it. The focus state remains. I haven't dive into why.

Reply:
> I haven't dive into why.

You shouldn't. I'm proposing the change so I should fix it.

Reply:
I can think of two possible paths: one is in the core (chain callback), one is in the demos (only forward the /aria-/). I'm wondering if the latter isn't the best one.

Reply:
We need to re-evaluate that component (InputBase). There's too much going on there with props filtering i.e. what prop is passed to which component. Some changes might fall under bug or breaking change depending on the viewpoint. Will follow up on this.

Reply:
In this case, should we remove the change of the auto complete from this pull request so we can move forward with the others? (I need to look at the other changes).

Reply:
What's wrong with the current approach? Focus handling should work now as of 4c0e285312169b4fb818e7f97d52f64ca2b4befe